p256/Cargo.toml

```diff
@@ -50,7 +50,7 @@ hash2curve = ["arithmetic", "elliptic-curve/hash2curve"]
 jwk = ["elliptic-curve/jwk"]
 pem = ["elliptic-curve/pem", "ecdsa-core/pem", "pkcs8"]
 pkcs8 = ["ecdsa-core/pkcs8", "elliptic-curve/pkcs8"]
-serde = ["ecdsa-core/serde", "elliptic-curve/serde", "serdect"]
+serde = ["ecdsa-core/serde", "elliptic-curve/serde", "dep:primeorder", "serdect"]
```

Comment:
```suggestion
serde = ["ecdsa-core/serde", "elliptic-curve/serde", "primeorder/serde", "serdect"]
```

Reply:
@tarcieri  - I'm not sure that works; primeorder is optional itself (I tried your solution first, prior to the dep: change).

Reply:
ie, to be clear - your change won't compile for me ;)